templates/ceph/shared/_latest-news-ceph.html

```diff
@@ -0,0 +1,44 @@
+<section class="p-section--deep">
+  <div class="u-fixed-width">
+    <hr class="p-rule" />
+    <div class="p-section--shallow">
+      <h2>
+        <a href="{{ title_link }}">Latest news from&nbsp;our&nbsp;blog&nbsp;&rsaquo;</a>
```

Comment:
Are the `&nbsp;`'s needed here?

Reply:
Most of this template is copied from the /desktop version (layout was different so I couldn't reuse), but at the time of building that these were requested by design